service/controller/v3/resource/instance/create.go

```diff
@@ -6,18 +6,20 @@ import (
 
```

Comment:
Github decided to hide this file diff. It basically went unreviewed. This is why I was asking what this PR is about. After you messaged in #news-dev I checked again and found that.

It's shame.

I reviewed it again. Nothing but minor comments left. Safe to ignore.

Reply:
Wow. :D